Scripts/RemediationScripts/Remediate-SetSQLServerMinReqTLSVersion.ps1

```diff
@@ -0,0 +1,999 @@
+﻿<###
+# Overview:
+    This script is used to set required TLS version for SQL Server in a Subscription.
+
+# Control ID:
+    Azure_SQLDatabase_DP_Use_Secure_TLS_Version_Trial
+
+# Display Name:
+    Use Approved TLS Version in SQL Server.
+
+# Prerequisites:
+    1. Contributor or higher privileges on the SQL Servers in a Subscription.
+    2. Must be connected to Azure with an authenticated account.
+
+# Steps performed by the script:
+    To remediate:
```

Comment:
Looks like there is typo, 'minimal' instead it should be minimum

Reply:
For SQL it is coming as Minimal or Storage it is coming as Minimum